evm/src/main/java/org/hyperledger/besu/evm/MainnetEVMs.java

```diff
@@ -546,7 +547,7 @@ public static EVM berlin(final EvmConfiguration evmConfiguration) {
    * @return the evm
    */
   public static EVM berlin(final BigInteger chainId, final EvmConfiguration evmConfiguration) {
-    return berlin(new IstanbulGasCalculator(), chainId, evmConfiguration);
+    return berlin(new BerlinGasCalculator(), chainId, evmConfiguration);
```

Comment:
this makes sense, just wondering if changing this is going to break anything? it's been this way for a long time!

Reply:
It's broken and wrong.  Should have caught it earlier.  It doesn't align with what the real definition of berlin is.  Also, only impacts users of the relatively new fluent evm APIs.  One researcher found a differential fuzzing bug based off of it, and this was the bug.

Reply:
ok makes sense then